### PR TITLE
Add kwack - In-Memory Analytics for Kafka using DuckDB

### PR DESCRIPTION
**Pull request by rayokota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ You can chat with this page's content on [HuggingChat](https://hf.co/chat/assist
 - [Sqlite2Duckdb](https://github.com/dridk/sqlite2duckdb) - A CLI tool to convert SQLite database to DuckDB.
 - [nodbi](https://docs.ropensci.org/nodbi/) - NoSQL Database Connector for R, providing a common API across Elasticsearch, CouchDB, MongoDB, SQLite, PostgreSQL, and DuckDB.
 - [duckplyr](https://duckdblabs.github.io/duckplyr/) - Drop-in replacement for dplyr in R that uses DuckDB for performance.
+- [kwack](https://github.com/rayokota/kwack) - In-Memory Analytics for Kafka using DuckDB
```
Comment (davidgasquez) [inline README.md:158]:
```suggestion
- [kwack](https://github.com/rayokota/kwack) - In-Memory Analytics for Kafka using DuckDB.
```

**Review by davidgasquez — CHANGES_REQUESTED** [review]
Thanks for the PR! 

Added a small suggestion to please the linter.

**Comment by rayokota** [comment]
Thanks @davidgasquez , committed your suggestion

**Review by davidgasquez — APPROVED** [review]
Sweet! Merging it now.

**Pull request merged**